go-backend/db/organization.go

```diff
@@ -50,32 +54,40 @@ const (
 		hi5_limit,
 		hi5_quota_renewal_frequency,
 		timezone FROM organizations ORDER BY name ASC`
-	emailRegexString = `^([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9_\-\.]+)\.([a-zA-Z]{2,5})$`
+
+	getOrganizationByDomainNameQuery = `SELECT * FROM organizations WHERE domain_name=$1 LIMIT 1`
+	getOrganizationByIDQuery         = `SELECT * FROM organizations WHERE id=$1 LIMIT 1`
```

Comment:
@jahio not sure but I think its not a good practice to use `SELECT *`. Instead we can specify the required fields here
https://stackoverflow.com/questions/321299/what-is-the-reason-not-to-use-select

Reply:
This is intended to be used for selecting a domain name in a lot of various different contexts, which we won't necessarily know what their needed fields might be ahead of time. This also could be premature optimization here. If we had like 100+ different fields then yeah we'd want to limit that result set, but we have far, far less than that.